integration-tests/regression.test.js

```diff
@@ -1,11 +1,16 @@
-const { getByText, queryAllByText, fireEvent, waitFor, getByLabelText } = require('@testing-library/dom');
+const { getByText, queryAllByText, fireEvent, waitFor, getByLabelText, queryByText } = require('@testing-library/dom');
 const { default: userEvent } = require('@testing-library/user-event');
-const { startApp } = require('./test-app');
+const { startAppAndWait } = require('./test-helpers');
```

Comment:
Using `startAppAndWait` here instead of `startApp` - while not all tests need this, I'd like to slowly start making this the standard. See `test-helpers.ts` to understand what this does.